### DISCUSSION TITLE
AAPS

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2078] [Q&A]
@MilosKozak 
It comes up on facebook frequently that AAPS advises people not to use xDrip as a collector.

Is there a reason for that?
Is there something we can do to xDrip to make it compatible?

I cannot promise it will be done.  But, if we don't know what it is, we certainly cannot do it.
Would you please provide any related insight you may have?

Thanks

**Comment by Der-Schubi** [comment]
I'm a little late, but this might has to do with the AAPS 3.0 upgrade guide in the docs: https://androidaps.readthedocs.io/en/latest/Installing-AndroidAPS/Releasenotes.html
"if you are using G5/G6 and xDrip as a collector, you have to update xDrip to a nightly version newer than 14th January 2022
if you are using G5/G6 switching to BYODA as collector is recommended to take advantage of back-smoothing (you can still use xDrip for other purposes, xDrip can receive data from BYODA)"
